src/components/sections/Cards/RitcherCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+type Level = {
+  id: string;
+  range: string;
+  label: string;
+  description: string;
+  color: string;
+};
```

Comment:
1. Rename the folder from `Cards` to `ritcher-scale-section`
2. Under `components/widgets` create a new folder called `ritcher-card`
3. Move this type declaration to a `model.ts` file under the `ritcher-card` folder